test/integration/private_connection/controls/forseti.rb

```diff
@@ -0,0 +1,180 @@
+    its('members') { should include "serviceAccount:#{forseti_server_service_account}" }
+  end
+  describe google_project_iam_binding(project: project_id, role: "roles/storage.objectCreator") do
+    it { should exist }
+    its('members') { should include "serviceAccount:#{forseti_server_service_account}" }
+  end
+  describe google_project_iam_binding(project: project_id, role: "roles/cloudsql.client") do
+    it { should exist }
+    its('members') { should include "serviceAccount:#{forseti_server_service_account}" }
+  end
+  describe google_project_iam_binding(project: project_id, role: "roles/logging.logWriter") do
+    it { should exist }
+    its('members') { should include "serviceAccount:#{forseti_server_service_account}" }
+  end
+  describe google_project_iam_binding(project: project_id, role: "roles/iam.serviceAccountTokenCreator") do
+    it { should exist }
+    its('members') { should include "serviceAccount:#{forseti_server_service_account}" }
+  end
+
+  describe google_storage_buckets(project: project_id) do
+    its('bucket_names') { should include forseti_server_storage_bucket }
+    its('bucket_names') { should include(/forseti-cai-export/) }
+  end
+
+  describe google_storage_bucket_objects(bucket: forseti_server_storage_bucket) do
+
+    # Enumerate the files that we expect to be present. This fixture ensures that we
+    # don't silently drop a rules file.
+    let(:expected_files) do
+      %w[
+        rules/audit_logging_rules.yaml
+        rules/bigquery_rules.yaml
+        rules/blacklist_rules.yaml
+        rules/bucket_rules.yaml
+        rules/cloudsql_rules.yaml
+        rules/enabled_apis_rules.yaml
+        rules/external_project_access_rules.yaml
+        rules/firewall_rules.yaml
+        rules/forwarding_rules.yaml
+        rules/group_rules.yaml
+        rules/groups_settings_rules.yaml
+        rules/iam_rules.yaml
+        rules/iap_rules.yaml
+        rules/instance_network_interface_rules.yaml
+        rules/ke_rules.yaml
+        rules/ke_scanner_rules.yaml
+        rules/lien_rules.yaml
+        rules/location_rules.yaml
+        rules/log_sink_rules.yaml
+        rules/resource_rules.yaml
+        rules/retention_rules.yaml
+        rules/role_rules.yaml
+        rules/service_account_key_rules.yaml
+      ]
+    end
+
+    # Enumerate the files that are present in the rules directory  This fixture ensures
+    # that we don't miss an included rules file.
+    let(:present_files) do
+      template_dir = File.expand_path(
+        "../../../../modules/rules/templates/rules",
+        __dir__
+      )
+      Dir.glob("#{template_dir}/*.yaml").map {|file| "rules/#{File.basename(file)}" }
+    end
+
+    let(:files) { expected_files | present_files }
+
+    its('object_names') { should include(*files) }
+  end
+
+  describe google_service_account(project: project_id, name: forseti_server_service_account) do
+    its(:display_name) { should eq "Forseti Server Service Account" }
```

Comment:
current version of inspec-gcp changed parameters for the `google_service_account` and now requires `project` to be set separately and `name` to be the email of the service account.